web/client/epics/annotations.js

```diff
@@ -122,8 +122,8 @@ const validateFeatureCollection = (feature) => {
  * @param drawMethod
  * @return {boolean}
  */
-const getGeodesicProperty = (state, drawMethod = "Circle") => {
-    return drawMethod === "Circle" && get(state.annotations, "config.geodesic", false);
+const getGeodesicProperty = (state) => {
```

Comment:
this function is used in many parts in this file can you fix also their call, we are no longer using a second parameter.

Reply:
Done